addon/components/au-checkbox.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class AuCheckboxComponent extends Component {
+  get groupValue() {
+    return this.args.groupValue || [];
+  }
+
+  get isCheckedInGroup() {
+    const { name } = this.args;
+    return this.groupValue.includes(name);
+  }
+
+  get checked() {
+    const { inGroup, checked } = this.args;
+    return inGroup ? this.isCheckedInGroup : checked;
+  }
+
+  @action
+  onChange(event) {
+    if (this.args.isDisabled === true) {
+      return;
+    }
```

Comment:
I'm not sure if this is needed. I don't think browsers trigger the event if the input is disabled?

Also, I think it needs to be `this.args.disabled` in our version and we can also remove the `=== true`?

```suggestion
    if (this.args.disabled) {
      return;
    }
```

Reply:
I saw this within the Velvet implementation but I was also in doubt this should be added/copied. Removed ✅